apiary.apib

```diff
@@ -1037,6 +1104,135 @@ Available types are:
 - insuranceHistory: InsuranceHistoryItem (array[InsuranceHistoryItem], required) - All insurance intervals
 - smpPrice: 123.45 (number) - Price for which this investment can be offered on SMP. If canBeOffered flag is true but this field is null it means that price is available only on request to smpSellInfo. Same rules apply for smpFee and smpFeeExpirationDate fields
 
+## InvestmentSummary (object)
+
+- investmentsCount: `450` (number, required) - Number of investments
+- principalRemaining: `67038.00` (number, required) - Sum of remaining principals
+- principalPaid: `34251.02` (number, required) - Sum of paid principals
+- interestRemaining: `6768.52` (number, required) - Sum of remaining interests
+- interestPaid: `13580.28` (number, required) - Sum of paid principals
+
+## InvestmentDetail (object)
+
+- id: `628265` (number, required) - Id of investment
+- loan (object, required)
+    - id: `56321` (number, required) - Id of loan
+    - activeLoanOrdinal: `1` (number, required) - Number of *other* active loans of the same borrower
+    - contractNo: `1007376013` (string, required) - Immutable number identifying loan across Zonky platform
+    - title: `Restrukturalizace úvěrů` (string, required) - Title or name of loan
+    - story: `Dlužím, kam se podívám, a chci to splatit.` (string, required) - Loan story
+    - annuity: `1400` (number, required) - annuity (monthly repayment)
+    - labels: [`VERIFIED_INCOME`] (array[enum], fixed-type) - additional information about loan status
+        - Members
+            - `VERIFIED_BORROWER`
+            - `VERIFIED_INCOME`
+            - `CURRENTLY_INSURED`
+            - `ADDITIONALLY_INSURED`
+            - `BLACK_FRIDAY`
+            - `COVID_19_REQUEST_WAITING`
+            - `COVID_19_POSTPONEMENT_PROCESSED`
+    - borrower (object, required)
+        - id: `125646` (number, required) - borrower internal id
+        - primaryIncomeType: `EMPLOYMENT` (enum, required) - borrower primary income type
+            - Members
+                - `EMPLOYMENT`
+                - `SELF_EMPLOYMENT`
+                - `MATERNITY_LEAVE`
+                - `ENTREPRENEUR`
+                - `STUDENT`
+                - `PENSION`
+                - `LIBERAL_PROFESSION`
+                - `UNEMPLOYED`
+                - `OTHER`
```

Comment:
FYI I have previously seen `OTHERS_MAIN` here as well.

Reply:
I will check it again, but I believe that `OTHERS_MAIN` is not a valid option and was returned by some loans as a result of a bug